### PR TITLE
use record opening date label

### PR DESCRIPTION
**Pull request by colinbowen**
## Changes in this PR
- Remove Closure Expiry date and use record opening date
- Added tests for closure expiry date combinations

## JIRA ticket
https://national-archives.atlassian.net/browse/AYR-714

### Before
Screenshot 2024-02-13 at 18 30 04

### After

Screenshot 2024-02-13 at 18 27 59

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

**Review by anthonyhashemi — APPROVED** [review]
Nice one

File: app/templates/main/browse-consignment.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 {% set sorting_list = {
     "Record status (closed first)": "closure_type-asc",
     "Record status (open first)": "closure_type-desc",
-    "Closure expiry date (soonest first)": "closure_expiry_date-asc",
+    "Record opening date (soonest first)": "opening_date-asc",
     "Date last modified (most recent first)": "date_last_modified-desc",
     "Date last modified (oldest first)": "date_last_modified-asc",
     "Record filename (A to Z)": "file_name-asc",
```
Comment (pgandhizaizi) [inline app/templates/main/browse-consignment.html:13]: Please replace 'Date last modified' labels with 'Record date' as per terry's design and replace corresponding test cases too

**Comment by pgandhizaizi** [comment]
noticed all unit testing is using date_last_modified as date filter, it would be good if one or two test cases being added for 'record opening date' as date filter and sorting

**Comment by anthonyhashemi** [comment]
> noticed all unit testing is using date_last_modified as date filter, it would be good if one or two test cases being added for 'record opening date' as date filter and sorting

As mentioned above I dont think this PR is concerned with the label updates, just the closure consolidation.

If there were any tests for closure start and closure period, those should be updated to record opening date equivalent.

File: app/templates/main/browse-consignment.html (hunk 1 above)
Comment (anthonyhashemi) [reply app/templates/main/browse-consignment.html:13]: I think this PR is just for the combination of closure start date and period into record opening date

The label changes Terry wants will happen in another PR/ ticket I believe

Comment (colinbowen) [reply app/templates/main/browse-consignment.html:13]: Label changes are to be done in AYR-718

**Comment by colinbowen** [comment]
> noticed all unit testing is using date_last_modified as date filter, it would be good if one or two test cases being added for 'record opening date' as date filter and sorting

Added tests to check that results are sorted by opening date and filters

**Review by anthonyhashemi — APPROVED** [review]

**Pull request merged**